python/pyarrow/tests/test_fs.py

```diff
@@ -1303,12 +1303,12 @@ def test_s3_proxy_options(monkeypatch, pickle_module):
     # Missing port
     with pytest.raises(KeyError):
         S3FileSystem(proxy_options={'scheme': 'http', 'host': 'localhost'})
-    # Invalid proxy URI (invalid scheme htttps)
+    # Invalid proxy URI (invalid scheme httpsB)
```

Comment:
all those `httpsB` and `httpA` seem incorrect.

Reply:
The original content was buggy -- a comment referenced the wrong thing.

The content is trying to test cases where an invalid protocol is used, using typos for this purpose isn't a great strategy as it mixes up things and encourages people to ignore typos. By using a valid protocol + a capital letter, a code aware spell checker can catch unintentional spelling errors.